src/components/Button/Button.styled.ts

```diff
@@ -47,39 +43,28 @@ export const ButtonWrapper = styled.span`
     padding-bottom: 4px;
   }
 
-  &.normalButton.size_large.variant_outlined > button {
-    padding-top: 7px;
-    padding-bottom: 7px;
-  }
-
   &.normalButton.size_small > button .icon_left {
     margin-right: 4px;
     margin-left: -4px;
+		font-size: 14px;

```

Comment:
Everything looks good, the only thing that should be changed is this, it should be 16px:
"When we render an icon inside the button (not the icon button variant but when the button has an icon and text string), the icon should have a 16px font-size. This will shrink the icon in the button a little bit from it’s current point. It should be 16px in both small and medium."